starter-kits/product/index.rst

```diff
@@ -1,2 +1,2 @@
 .. datatemplate:yaml:: _data.yaml
-   :template: product-v1.rst
+   :template: product-v2.rst
```

Comment:
In another branch have you created a new 'v2' template? In that case, you should change this to 'v2' in your other branch and keep this as 'v1' in this branch

Reply:
I'm going to scrap the other branch, so this will be v2.